apps/gateway/lib/gateway/controllers/arena/currency_controller.ex

```diff
@@ -0,0 +1,25 @@
+defmodule Gateway.Controllers.Users.CurrencyController do
+  use Gateway, :controller
+
+  alias GameBackend.Utils
+  alias GameBackend.Users.Currencies.UserCurrency
+  alias GameBackend.Users.Currencies
+
+  @moduledoc """
```

Comment:
Why the moduledoc here? It looks so weird 😂

Reply:
ahahhahahaha i didn't noticed! moved